cmd/composectl/cmd/update/init.go

```diff
@@ -0,0 +1,70 @@
+package updatectl
+
+import (
+	v1 "github.com/foundriesio/composeapp/pkg/compose/v1"
+	"github.com/foundriesio/composeapp/pkg/update"
+	"github.com/schollz/progressbar/v3"
+	"github.com/spf13/cobra"
+)
+
+var (
+	initCmd = &cobra.Command{
+		Use:   "init [app_ref]...",
+		Short: "Initialize the update for specified apps by identifying required blobs to fetch",
+		Long:  `Initialize or reinitialize an update for the specified apps by determining which blobs need to be downloaded to fetch the update`,
```

Comment:
This might be a good place to include an `Example:` for the user.

Reply:
OK, will do.

Reply:
59a7be5